src/gz/gz_col_view.c

```diff
@@ -734,7 +764,7 @@ static void release_mem(void *p_ptr)
 
 void gz_col_view(void)
 {
-  const int dyn_poly_cap = 0x1000;
+  const int dyn_poly_cap = 0x1400;
```

Comment:
Is there any evidence to indicate that this buffer size needs to be increased? Memory is becoming more and more of a concern with the growing size of gz, and it's reached a point where I don't want to make these kinds of decisions on a whim.

Reply:
My initial thoughts were that since each waterbox took `(sizeof(Vtx) * 4 + sizeof(Gfx) * 2) * 5 = 0x190` bytes in the gfx buffer it would be needed for rooms with many waterboxes such as water temple or bottom of the well. However testing in game seems to suggest this is unnecessary after all. Will keep it at `0x1000`.